.github/workflows/on_bundle_update_available.yaml

```diff
@@ -28,9 +30,5 @@ jobs:
             Update bundles with new revisions from Charmhub.
           labels: |
             automated pr
-          assignees: team-reviewers, marceloneppel, dragomirp, taurus-forever
-          reviewers: team-reviewers, marceloneppel, dragomirp, taurus-forever
-          team-reviewers: |
-            owners
-            maintainers
+          reviewers: canonical/data-platform-postgresql
```

Comment:
Is it possible to add the team twice? It seems that once I accepted this review, the others were removed.

Reply:
@marceloneppel can we merge this to monitor how it flies over the time and revert if we are not happy?

Reply:
Yes we can merge and see if it happens again. I think I was notified after the first review, but let's merge and confirm in the next PR.